admin/views/tabs/social/facebook.php

```diff
@@ -40,9 +40,26 @@
 	$yform->textinput( 'og_frontpage_desc', __( 'Description', 'wordpress-seo' ) );
 
 	// Offer copying of meta description.
-	echo '<input type="hidden" id="meta_description" value="', WPSEO_Options::get( 'metadesc-home-wpseo' ), '" />';
-	echo '<p class="label desc"><a href="javascript:;" onclick="wpseoCopyHomeMeta();" class="button">', esc_html__( 'Copy home meta description', 'wordpress-seo' ), '</a></p>';
+	$homepage_meta_description = WPSEO_Options::get( 'metadesc-home-wpseo' );
+	if ( ! empty( $homepage_meta_description ) ) {
+		$copy_home_meta_desc_help = new WPSEO_Admin_Help_Panel(
+			'copy-home-meda-desc',
+			esc_html__( 'Help on copying the home meta description', 'wordpress-seo' ),
+			sprintf(
+				/* translators: 1: link open tag; 2: link close tag. */
+				esc_html__( 'Click the "Copy home meta description" button to use the meta description already set in the %1$sSearch Appearance Homepage%2$s setting.', 'wordpress-seo' ),
+				'<a href="' . esc_url( admin_url( 'admin.php?page=wpseo_titles' ) ) . '">',
```

Comment:
This should be `admin.php?page=wpseo_titles#top#home` as the meta description is present on the `home` tab instead of the first default tab.

Reply:
@moorscode sorry on which branch there's a Home tab? For me, in both trunk and release/7.0 the homepage setting is on the General tab.

Reply:
You are right, that is weird.